src/Views/IframeView.php

```diff
@@ -168,6 +168,31 @@ public function setFormId( $id ) {
 		return $this;
 	}
 
+	/**
+	 * Return whether or not rest api request to render donation form block.
+	 *
+	 * @return bool
+	 * @since 2.7.0
+	 */
+	private function isDonationFormBlockRendererApiRequest() {
+		return false !== strpos( $_SERVER['REQUEST_URI'], rest_get_url_prefix() . '/wp/v2/block-renderer/give/donation-form' );
```

Comment:
@DevinWalker I research about few ways to detect if we rendering donation form bock or not but I did not find anything solid.  This is a quick hack to detect if we are rendering donation form block or not.